app/components/registration-photos/template.hbs

```diff
@@ -0,0 +1,33 @@
+<div class='collapse' id='collapseExample'>
+  <div class='card card-body'>
+    <div>
+      <input
+        aria-label='bird1'
+        type='image'
+        class='register-photo'
+        src='/assets/images/imagesForUserRegistration/owl.jpg'
+        alt='owl'
+        value={{'owl.jpg'}}
```

Comment:
When you pass property like `string` don't use curly brackets for better readability:
```suggestion
        value='owl.jpg'
```